docker/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM continuumio/miniconda3:4.8.2
```

Comment:
How about using one of the [jupyter-docker-stacks](https://jupyter-docker-stacks.readthedocs.io/en/latest/) as a base? This has a robust entry point for running jupyter, and more.

Reply:
Would `jupyter/base-notebook` be the most appropriate?

Reply:
@banesullivan yes, `jupyter/base-notebook` is a good idea

Reply:
Testing out with `jupyter/base-notebook`

Reply:
We're now using `jupyter/base-notebook:python-3.7.6`, and the example work from the container.